ui/src/app/applications/components/application-resource-tree/application-resource-tree.tsx

```diff
@@ -603,6 +604,61 @@ function expandCollapse(node: ResourceTreeNode, props: ApplicationResourceTreePr
     props.setNodeExpansion(node.uid, isExpanded);
 }
 
+function NodeInfoDetails({tag: tag, kind: kind}: {tag: models.InfoItem; kind: string}) {
+    if (kind === 'Pod') {
```

Comment:
Check when pod parent resource is Job type. It should surface the right container status

```
name: create
      ready: false
      restartCount: 0
      started: false
      state:
        terminated:
          containerID: >-
            docker://d8bf084916ace4925e3db86c859a3ff4fe7d16b104e7ab75b13cb00523366cc3
          exitCode: 0
          finishedAt: '2023-03-11T05:12:42Z'
          reason: Completed
          startedAt: '2023-03-11T05:12:41Z'
```